### DISCUSSION TITLE
How to set Ring Tone on Android 14

### DISCUSSION THREAD

**Discussion by jssilva25** [baresip/baresip#3142] [Q&A] [answered]
I have BareSip simple perfectly working with an asterisk server for a couple of months with complete satisfaction, always connected, no lost calls. Thank you to the devs.

My only problem is, during all this time I've been struggling with my settings to find the place where I can set another Ring Tone because the default is almost inaudible. I can reset the notifications but not the Ring Tone.

Can anybody help, please?

**Comment by juha-h** [comment]
If you are using baresip Android app, it is using whatever the default
ringtone on your device is.  You can change the default ringtone in Android
settings.

**Comment by jssilva25** [comment]
> If you are using baresip Android app, it is using whatever the default
ringtone on your device is

Thank you for replying. I wish that to be true but, in my case, it is not. My regular Ring Tone is one, which in fact I've reset, but the baresip one is not the same, and I can't change it as I said before.

Furthermore, I can change the regular Ring Tone and it has no effect on the baresip one.

**Comment by juha-h** [comment, marked as answer]
jssilva25 writes:

> > If you are using baresip Android app, it is using whatever the default
> ringtone on your device is
> 
> Thank you for replying. I wish that to be true but, in my case, it is
> not. My regular Ring Tone is one, which in fact I've reset, but the
> baresip one is not the same, and I can't change it as I said before.

Just tested and changing default ringtone also changed it in baresip
app.  But you need to restart baresip after the change.

**Reply by jssilva25** [reply]
Thank you, it's working now, but there's something fishy here:

I reset the phone ring tone to the default (Mi Remix), called the voip baresip number and it didn't ring, didn't even display the call input. Tried with another number that is forwarded to baresip, same behavior. Called the phone and it rung Remix tone.

Then reset the phone to the previous ring tone and baresip started working with this same ring tone.

It's strange but for me it's fixed, the comments are for the devs. I'm available for further tests if needed. The phone is a Xiaomi Redmi 13 Pro 5G.

Thank you again.
